### PR TITLE
Fix typo assesment -> assessment

### PR DESCRIPTION
**Pull request by stzn**
Bug/issue #, if applicable: 

## Summary

I found typos in some places, so fixed them. please correct me if I am wrong.

## Dependencies

None

## Testing

Please check the spelling

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [ ] ~Added tests~
- [x] Ran the `./bin/test` script and it succeeded
- [ ] ~Updated documentation if necessary~

### CODE REVIEW

**Comment by ethan-kusters** [comment]
@swift-ci please test

File: Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json [hunk 1]
```diff
@@ -1936,7 +1936,7 @@
                     },
                     "sections": {
                         "type": "string",
-                        "enum": ["task", "assesment", "heading"]
+                        "enum": ["task", "assessment", "heading"]
                     }
```
Comment (Kyle-Ye) [inline Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json:1940]: Is this a breaking change for swift-docc-render? @ethan-kusters

**Comment by Kyle-Ye** [comment]
@swift-ci please test

**Review by Kyle-Ye — CHANGES_REQUESTED** [review]
Would you mind writing a test case for it before merging?

File: Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json (hunk 1 above)
Comment (Kyle-Ye) [reply Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json:1952]: The is confirmed on swift-docc-render and swift-docc

**Comment by stzn** [comment]
I see. Thank you. What kind of tests do I need to add? Sorry, it,s my first time contribution and not sure what to do actually.

**Comment by Kyle-Ye** [comment]
It corresponds to the code in `RenderHierarchyLandmark.swift`.

After searching in project, it turns out that we currently do not have any sort of test case for openapi json file. Maybe we can add such later when swift-openapi-generator is mature.

**Review by Kyle-Ye — APPROVED** [review]

**Comment by Kyle-Ye** [comment]
@swift-ci please test

**Pull request merged**